cypresse2e/cypress/integration/e2eTests/privacyPolicy/privacyPolicy.spec.js

```diff
@@ -0,0 +1,261 @@
+    contentPara.find("li").each(($el, i) => {
+      switch (i) {
+        case 0:
+          if (firstUTxt != "") {
+            expect($el.find("u").text()).to.eq(firstUTxt);
+          }
+          if (firstLTxt != "") {
+            expect($el.text()).to.include(firstLTxt);
+          }
+          break;
+        case 1:
+          if (secondUTxt != "") {
+            expect($el.find("u").text()).to.eq(secondUTxt);
+          }
+          if (secondLTxt != "") {
+            expect($el.text()).to.include(secondLTxt);
+          }
+          break;
+        case 2:
+          if (thirdUTxt != "") {
+            expect($el.find("u").text()).to.eq(thirdUTxt);
+          }
+          if (thirdLTxt != "") {
+            expect($el.text()).to.include(thirdLTxt);
+          }
+          break;
+        case 3:
+          if (fourthUTxt != "") {
+            expect($el.find("u").text()).to.eq(fourthUTxt);
+          }
+          if (fourthLTxt != "") {
+            expect($el.text()).to.include(fourthLTxt);
+          }
+          break;
+        case 4:
+          if (fifthUTxt != "") {
+            expect($el.find("u").text()).to.eq(fifthUTxt);
+          }
+          if (fifthLTxt != "") {
+            expect($el.text()).to.include(fifthLTxt);
+          }
+          break;
+        case 5:
+          if (sixthhUTxt != "") {
+            expect($el.find("u").text()).to.eq(sixthhUTxt);
+          }
+          if (sixthLTxt != "") {
+            expect($el.text()).to.include(sixthLTxt);
+          }
+          break;
+        case 6:
+          if (seventhhUTxt != "") {
+            expect($el.find("u").text()).to.eq(seventhhUTxt);
+          }
+          if (seventhLTxt != "") {
+            expect($el.text()).to.include(seventhLTxt);
+          }
+          break;
+        default:
+          break;
+      }
+    });
+    //check on the next div
+    if (secondDivTxt != "")
+      contentPara.parents("div").next("div").contains(secondDivTxt);
+  }
+
+  //to get object data
+  function getObjectData(dataType, ParaNum) {
+    var objArray;
+    var objCx;
+    var returnObjCx = [];
+    switch (dataType) {
+      case "paraPolicyCx":
+        objArray = privatePolicyContent.paraPolicyCx;
+        break;
+      case "paraPolicyCxWithOrdList":
+        objArray = privatePolicyContent.paraPolicyCxWithOrdList;
+        break;
+
+      case "paraPolicyCxOrdListWthnOrdList":
+        objArray = privatePolicyContent.paraPolicyCxOrdListWthnOrdList;
+        break;
+      case "paraPolicyCxDiffTyp1":
+        objArray = privatePolicyContent.paraPolicyCxDiffTyp1;
+        break;
+      case "paraPolicyCxDiffTyp2":
+        objArray = privatePolicyContent.paraPolicyCxDiffTyp2;
+        break;
+      default:
+        break;
+    }
+    objArray.forEach(function (objCx) {
+      if (objCx.id == ParaNum) {
+        returnObjCx = objCx;
+      }
+      if (returnObjCx.length > 0) return returnObjCx;
+    });
+    return returnObjCx;
+  }
+});
```

Comment:
Will cases be written for paragraphs 4, 6-8? 

Do you think it's necessary to also test if the email links in the policy are clickable?

Reply:
@jessica-lau - Yes, for paragraphs 4, 6,  8 , I will work on it next week and also include to test the email links for clickable. TY

Reply:
@jessica-lau @adorapham 
Do you think it's necessary to test all paragraphs? Or maybe to check only the email links?

Reply:
Hi @adorapham 
I think the tests for email links and FP logo missing.
Thanks

Reply:
@galeza @jessica-lau i think if testing email is not safe enough because i think the content should be also important, but testing everything might be too much. How is about just testing one item in each paragraph to make sure the paragraph is existing?

Reply:
@adorapham it should be like the below. PrivacyPolicy first letter should be capitalized because class name in the POM should start with a capitalized letter and then it's imported into the spec. The json file can remain camelcased.

import PrivacyPolicy from "../../../elements/pages/privacyPolicy";
import privacyPolicyContent from "../../../fixtures/privacyPolicyContent.json";